Src/Shared/Core/Active.cpp

```diff
@@ -88,14 +88,14 @@ void CActive::CheckForAndThrowException() const
 {
 	if(_isDone)
 	{
-		if(_activeException)
+		if (std::exception_ptr exception = _activeException)
```

Comment:
not quite the same as what's coming in C++.

It's actually a special case in the standard:
http://eel.is/c++draft/stmt.select#1